src/content/staking/solo/index.md

```diff
@@ -0,0 +1,175 @@
+  </ExpandableCard>
+</InfoGrid>
+
+<StakingComparison page="solo" />
+
+## How it works {#how-it-works}
+
+<HowSoloStakingWorks />
+
+If ever desired, you can exit as a validator which eliminates the requirement to be online, and stops any further rewards. Be aware that until the planned Shanghai upgrade _withdrawing_ those funds will not be possible.
+
+After Shanghai, users will be able to withdraw their rewards as well as their stake if they choose.
+
+## Get started on the Staking Launchpad {#get-started-on-the-staking-launchpad}
+
+The Staking Launchpad is an open source application that will help you become a validator. It walks you through a preparation checklist, how to choose clients, generate your keys and assists with depositing your ETH to the staking deposit contract.
+
+<LaunchpadWidget />
+
+## What to consider with node and client setup tools {#node-tool-considerations}
+
+There are a growing number of tools and services to help you solo stake your ETH, but each come with different risks and benefits.
+
+Attribute indicators are used below to signal notable strengths or weaknesses a listed staking tool may have. Use this section as a reference for how we define these attributes while you’re choosing what tools to help with your staking journey.
+
+<StakingConsiderations page="solo" />
+
+## 🛠 Explore node and client setup tools {#node-and-client-tools}
+
+There are a variety of options available to help you with your setup. Use the above indicators to help guide you through the tools below.
+
+<Emoji text="⚠️" mr="1rem" /> Please note the importance of choosing a <a href="/developers/docs/nodes-and-clients/client-diversity/">minority client</a> as it improves the security of the network, and limits your risk. Tools that allow you to setup minority client are denoted as "multi-client."
+
+<StakingProductsCardGrid category="nodeTools" />
+
+**Key Generators**
+
+These tools can be used as an alternative to the [Staking Deposit CLI](https://github.com/ethereum/staking-deposit-cli) to help with key generation.
+
+<StakingProductsCardGrid category="keyGen" />
+
+Have a suggestion for a staking tool we missed? Check out our [product listing policy](/contributing/adding-staking-product) to see if it would be a good fit, and to submit it for review.
+
+## 📖 Explore solo staking guides {#staking-guides}
+
+- [CoinCashew's Ethereum 2.0 Guide](https://www.coincashew.com/coins/overview-eth/guide-or-how-to-setup-a-validator-on-eth2-mainnet) - Linux (CLI)
+- [Somer Esat](https://github.com/SomerEsat/ethereum-staking-guide) - Linux (CLI)
+- [Rocket Pool Node Operators](https://rocketpool.net/node-operators) - Linux, macOS (CLI)
+
+<!-- TODO: beaconcha.in/education page -->
+
+## FAQ {#faq}
+
+These are a few of the most common questions about staking that are worth knowing about.
+
+<ExpandableCard title="What is a validator?">
```

Comment:
Great FAQ - I think we should also list this on the staking homepage given we mention the term multiple times there.

Reply:
@samajammin We're lacking FAQ content on the landing page at the moment... think we should we just duplicate this question? Or move it from the solo page to the landing page?